package.json

```diff
@@ -21,7 +21,7 @@
     "clean": "rimraf build",
     "lint": "eslint src test",
     "prepublish": "npm run clean && npm run build",
-    "test": "mocha --compilers js:babel-register --recursive --require ./test/setup.js",
+    "test": "mocha --compilers js:babel-register --recursive ./test/components/index.jsx --require ./test/setup.js",
```

Comment:
It's just a quick fix. What if I want to test another component as a part of the whole project?

Reply:
Yes, it was rapid solvement to launch test. I believe i'll make it right.

Reply:
Deal